src/olympia/devhub/templates/devhub/includes/addon_details.html

```diff
@@ -94,21 +94,23 @@
   </li>
 {% endif %}
 {% set latest_listed_version=addon.find_latest_version(channel=amo.RELEASE_CHANNEL_LISTED) %}
-{% if latest_listed_version and latest_listed_version != addon.current_version %}
-  <li>
-    <strong>{{ _('Next Version:') }}</strong>
-    {{ link_if_listed_else_text(latest_listed_version, latest_listed_version.version) }}
-    <span class="tip tooltip" title="{{ _('This is the newest uploaded version, however it isn’t live on the site yet.') }}">?</span>
-  </li>
-{% endif %}
-{% with position = get_position(addon) %}
-  {% if position and position.pos and position.total %}
-    <li class="queue-position" title="{{ _('Queues are not reviewed strictly in order') }}">
-      <strong>{{ _('Queue Position:') }}</strong>
-      {% trans position=position.pos|numberfmt,
-               total=position.total|numberfmt %}
-        {{ position }} of {{ total }}
-      {% endtrans %}
+{% if addon.is_listed %}
```

Comment:
I don't understand this change - we're adding back a check for `addon.is_listed`??  If there isn't a listed version then `{% if latest_listed_version ... ` is going to be false, and if there aren't any listed versions there won't be any in the queue either.
If you're hiding the queue check for perf reasons you can use `Addon.has_listed_versions`.